submissions/sadoharu/task_js_dom_/scripts.js

```diff
@@ -0,0 +1,61 @@
+window.addEventListener("DOMContentLoaded", () => {
+  const pages = document.querySelectorAll(".section__page");
+  const navList = document.querySelector(".nav-bar__list");
+
+  const navListItem = [
+    {
+      title: "Overview",
+      icon: "icon-browser",
+    },
+    {
+      title: "About",
+      icon: "icon-about-dot-me",
+    },
+    {
+      title: "Achievement",
+      icon: "icon-award",
+    },
+    {
+      title: "Projects",
+      icon: "icon-files-empty",
+    },
+    {
+      title: "Reviews",
+      icon: "icon-rate_review",
+    },
+  ];
+
+  navListItem.forEach((item, i) => {
+    const liItem = document.createElement("li");
+    liItem.classList.add("nav-bar__list_item");
+    liItem.setAttribute("data-index", i);
+    liItem.innerHTML = `
+      <button class="nav-bar__list_button desktop">
+        <span class="${item.icon}">
+        </span>
+        ${item.title}
+      </button>
+      <button class="nav-bar__list_button mobile ${item.icon}"></button>
+    `;
+    navList.append(liItem);
```

Comment:
You're reaching out to dom for every button to add it to the list. Find a way to make a single action to add all buttons at once. [hint](https://developer.mozilla.org/ru/docs/Web/API/Document/createDocumentFragment)